basic-map.ipynb

```diff
@@ -0,0 +1,82 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "width": 500
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Import the PyGMT library\n",
+    "import pygmt\n",
+    "\n",
+    "region=[127.5, 128.5, 26, 27]\n",
+    "projection=\"M10c\"\n",
+    "\n",
+    "\n",
+    "# Use the Figure class to create a figure object, named \"fig\"\n",
+    "fig = pygmt.Figure()\n",
+    "fig.coast(\n",
+    "    region=region,\n",
+    "    projection=projection,\n",
```

Comment:
Looks like that `region` and `projection` are both used only once. Thus, I feel we do not need to define variables, but can pass the arguments directly to the parameters.

```suggestion
    "import pygmt\n",
    "\n",
    "\n",
    "# Use the Figure class to create a figure object, named \"fig\"\n",
    "fig = pygmt.Figure()\n",
    "fig.coast(\n",
    "    # Set the region of the plot\n",
    "    # Format is [xmin, xmax, ymin, ymax]\n",
    "    region=[127.5, 128.5, 26, 27],\n",
    "    # Set the projection to \"M\" for \"Mercator\"\n",
    "    # Specify the width as 10 centimeters\n",
    "    projection=\"M10c\",\n",
```

Reply:
Since the example section is demonstrating the Python-GMT syntax, I think it may be a good idea to show how we can assign data to a Python variable (a list for `region` and a string for `projection`) and then pass it to a PyGMT function. It doesn't have to be on this specific example, but I want to highlight why PyGMT offers Python-related advantages over GMT syntax.

Reply:
Ah, I see your point! From this perspective, assigning variables makes sense!